spec/lib/miq_automation_engine/service_methods/miq_ae_service_generic_object_spec.rb

```diff
@@ -53,6 +53,49 @@ module MiqAeServiceGenericObjectSpec
       end
     end
 
+    context '#convert_params_to_ar_model' do
```

Comment:
`describe` followed by a method name
`context` followed by a condition
`it' followed by a result